components/compass-runtime-agent/internal/kyma/model/types.go

```diff
@@ -101,7 +102,8 @@ type Credentials struct {
 	Oauth *Oauth
 	// BasicAuth configuration
 	Basic *Basic
-
+	// Additional request parameters
```

Comment:
The structure in Director API is as follows:
```
input AuthInput {
	credential: CredentialDataInput
	additionalHeaders: HttpHeaders @deprecated(reason: "Use `additionalHeadersSerialized`.")
	additionalHeadersSerialized: HttpHeadersSerialized
	additionalQueryParams: QueryParams @deprecated(reason: "Use `additionalHeadersSerialized`.")
	additionalQueryParamsSerialized: QueryParamsSerialized
	requestAuth: CredentialRequestAuthInput
}
```

Shouldn't we move `RequestParameters`  to `APIDefinition` type?

Reply:
The RequestParameters are already in api defitnion i will remove to from credentials

Reply:
I will check it the credentials are on the Auth object

                                "defaultInstanceAuth": {
                                    "credential": {
                                        "username": "user",
                                        "password": "pass"
                                    },
                                    "additionalHeaders": {
                                        "key1": [
                                            "value1"
                                        ],
                                        "key2": [
                                            "value2"
                                        ],
                                        "key3": [
                                            "value3"
                                        ]
                                    },
                                    "additionalQueryParams": null,
                                    "requestAuth": null
                                },